docs/examples/resnet50_trt/README.md

```diff
@@ -0,0 +1,255 @@
+# Triton Inference Server with DALI backend
+
+This repo is an example of [DALI (Data Loading Library) backend](https://github.com/NVIDIA/DALI) for image classification on [Triton Inference server](https://github.com/triton-inference-server/server).
+
+ResNet50 model optimized by [TensorRT](https://developer.nvidia.com/tensorrt) is used for image classification. 
+
+
+#### Dependencies
+
+* Export ONNX and build TensorRT
+  * `nvcr.io/nvidia/pytorch:20.12-py3`
+* Triton Inference Server for DALI backend
+  * `nvcr.io/nvidia/tritonserver:20.12-py3`
+  * `https://github.com/triton-inference-server/dali_backend`
```

Comment:
Maybe mention that this repo is just for the reference, and tritonserver docker image has DALI backend embedded.

Reply:
Done